### DISCUSSION TITLE
How to merge multiple DatasetDict

### DISCUSSION THREAD

**Discussion by MSZHabibie** [huggingface/datasets#5940] [Q&A] [answered]
Hi Everyone!!

I'm trying to merge 2 DatasetDict into 1 DatasetDict that has all the data from the 2 DatasetDict before. Here is my DatasetDict:

`DatasetDict({
    train: Dataset({
        features: ['audio', 'sentence'],
        num_rows: 46571
    })
})`

`DatasetDict({
    train: Dataset({
        features: ['audio', 'sentence'],
        num_rows: 2451
    })
})`

is there any way to merge DatasetDict? Thank You for your help

**Comment by mariosasko** [comment, marked as answer]
You can merge them as follows:

```python
DatasetDict({"train": datasets.concatenate_datasets([dd1["train"], dd2["train"]])})
```

**Reply by The-Rains** [reply]
Wow  its work, thank you so much for your help.
